images/reports/cloudbuild.yaml

```diff
@@ -1,17 +1,17 @@
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20200422-b25d964'
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221214-1b4dd4d69a'
     entrypoint: bash
     env:
       - TAG=$_GIT_TAG
       - BASE_REF=$_PULL_BASE_REF
       - REGISTRY=gcr.io/k8s-staging-ingressconformance
```

Comment:
this is wrong - should be "k8s-staging-ingress-controller-conformance"

Reply:
@thockin This is an image that was used to run in cloudbuild before that was hosted in `k8s-testimages` and now in the `k8s-staging-test-infra` project that is owned by the community.

this is used across several projects in the k8s org, and there is nothing related to the ingress conformance. it is just an image that has the basic things needed to run the cloudbuild job.

I did not follow your point :(

Reply:
oh,  I see https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/manifests/k8s-staging-ingress-controller-conformance/promoter-manifest.yaml says "gcr.io/k8s-staging-ingressconformance".  That's messed up.